docs/analytics.md

```diff
@@ -0,0 +1,56 @@
+# Analytics
+
+New Relic provides tools for application monitoring and web analytics. New Relic's [Application Performance Monitoring (APM)](https://docs.newrelic.com/docs/apm/) can monitor and measure application performance and activity on the backend. Their [Browser Agent](https://docs.newrelic.com/docs/browser/) can do the same for the frontend of the application.
+
+We can [create custom dashboards](https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) in New Relic that visualize the various metrics we collect, and combine metrics from the backend and frontend.
+
+## Metrics for Smart Hub
+
+### Version 1, 4/16/21
+
+#### Operational
+
+- % uptime
+
+- Average Response Time
+
+- Request rates
+
+- Error rates
+
+- Application/CPU usage
+
+#### Usage (by region and cumulative)
+
+- \# of registered users (in admin tool or New Relic?)
+
+- \# of closed user accounts (proxy for regional TTA turnover? Also in DOC
+SmartSheets)
+
+- \# of unique visitors
+
+- \# of visits per unique user
+
+- \# of visits by role (specialist, manager, federal staff by region)
+
+- Avg time on site by user
+
+#### Tasks (by region and cumulative)
+
+- Avg time spent on each AR
```

Comment:
Should this metric include some more ways to slice the data? I think Average might hide useful information such as a cohort that has a very low time spent becuase they're still doing everything in word and then copy/pasting into the hub.

I know that there is the copy/paste proxy also, but I think a certain amount of copy/paste will be present no matter what, especially on the resources fields.